CHANGELOG.md

```diff
@@ -1,5 +1,3 @@
-# ember-cli-fastboot changelog
```

Comment:
Removing the title here makes updating much easier (the tooling can always prepend the new version info, instead of splicing it in)